src/App.tsx

```diff
@@ -1,109 +1,25 @@
 import React from 'react';
-
 import './App.scss';
-
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import usersFromServer from './api/users';
+import commentsFromServer from './api/comments';
+import { PreparedPost } from './types/PreparedPost';
+import { PostList } from './components/PostList';
+
+const preparedPosts: PreparedPost[] = postsFromServer.map(
+  (post) => ({
+    ...post,
+    user: usersFromServer
+      .find(({ id }) => id === post.userId) || null,
```

Comment:
let's decompose this logic
Move this to the separate function

Reply:
added func